lib/sanbase_web/graphql/resolvers/social_data_resolver.ex

```diff
@@ -155,10 +156,26 @@ defmodule SanbaseWeb.Graphql.Resolvers.SocialDataResolver do
 
   def social_dominance(
         _root,
-        %{slug: slug, from: from, to: to, interval: interval, source: source},
+        %{slug: _slug, from: _from, to: _to, interval: _interval, source: source} = args,
         _resolution
       ) do
-    SocialData.social_dominance(%{slug: slug}, from, to, interval, source)
+    source =
```

Comment:
```suggestion
    source = if source == :all, do: :total, else: :source
```
you can do it on 1 row also

Reply:
``mix format`` makes it 5 lines of code, wouldn't that be a problem if it were to change?

Reply:
`mix format` cannot change between the 2 formats - the one with do..end, and the one with a kw-list (`,do: ... , else: ...`)

Reply:
ah, didn't see that it was a different format, thanks